src/components/utils/mutations.ts

```diff
@@ -7,29 +7,23 @@
 export function isMutationBelongsToElement(mutationRecord: MutationRecord, element: Element): boolean {
   const { type, target, addedNodes, removedNodes } = mutationRecord;
 
-  /**
-   * In case of removing the whole text in element, mutation type will be 'childList',
-   * 'removedNodes' will contain text node that is not existed anymore, so we can't check it with 'contains' method
-   * But Target will be the element itself, so we can detect it.
-   */
-  if (target === element) {
+  if (element.contains(target)) {
     return true;
   }
 
   /**
-   * Check typing and attributes changes
+   * In case of removing/adding the element itself, mutation type will be 'childList' and 'removedNodes'/'addedNodes' will contain the element.
    */
-  if (['characterData', 'attributes'].includes(type)) {
```

Comment:
Am I understanding it right that `characterData` and `attributes` mutations covered by `element.contains(target)` statement? Docs are not clear

Reply:
Yes, `element.contains(target)` covers all mutations happened to the element or it's descendants with the only one exception - removing/adding the element itself, because in this case target will be the element's parent and the element itself would be inside 'removedNodes'/'addedNodes'.

Reply:
Added more explanation + fixed formatting.